plugins/modules/network_settings_intent.py

```diff
@@ -1727,22 +1774,7 @@ def update_reserve_pool(self, config):
             return
 
         # Check update is required
-        obj_params = [
-            ("name", "name"),
-            ("type", "type"),
-            ("ipv6AddressSpace", "ipv6AddressSpace"),
-            ("ipv4GlobalPool", "ipv4GlobalPool"),
-            ("ipv4Prefix", "ipv4Prefix"),
-            ("ipv4PrefixLength", "ipv4PrefixLength"),
-            ("ipv4GateWay", "ipv4GateWay"),
-            ("ipv4DhcpServers", "ipv4DhcpServers"),
-            ("ipv4DnsServers", "ipv4DnsServers"),
-            ("ipv6GateWay", "ipv6GateWay"),
-            ("ipv6DhcpServers", "ipv6DhcpServers"),
-            ("ipv6DnsServers", "ipv6DnsServers"),
-            ("ipv4TotalHost", "ipv4TotalHost"),
-            ("slaacSupport", "slaacSupport")
-        ]
+        obj_params = self.get_obj_params("ReservePool")
```

Comment:
Can we init as reserve_pool_obj_params and pass it?

Same for global and network as well..